### ISSUE TITLE
Ordering by status on Scenarios Sequence page is not working

### ISSUE THREAD

**Issue by monofrei** [trivago/cluecumber-report-plugin#133]
Currently ordering by status on the Scenarios Sequence page is not working due to missing text in the cell.

1. Make sure to have Failed, Passed and Skipped scenarios in the report.
2. Open the Scenarios Sequence page.
3. Order the table by column Status
4. See behavior

**Expected behavior**
Order entire table by the scenarios' status: Failed, Passed, Skipped using natural alphabetic ordering.

**Additional context**
This could be fixed by changing the html code for the 'i' tag in common.ftl to wrap the status in a hidden span:
~~~~
 <i class="${class} cluecumber-icon icon-${icon}" data-toggle="tooltip"
       title="${status}"><span style="display:none">${status}</span></i>
~~~~
By this on the table will keep current style and the ordering will work.

**Comment by laxersaz** [comment]
Hi @monofrei !
Thanks a lot for reporting this!
Would you like to make a PR with your solution? If not, I will fix it for the next release.

**Comment by monofrei** [comment]
Hi @laxersaz, sure i will do it.

**Comment by laxersaz** [comment]
You can just fork the repo, make your changes and create a pull request from there.

**Comment by laxersaz** [comment]
Fixed in 1.7.1
